src/basic_recipes/errorbars.jl

```diff
@@ -1,13 +1,12 @@
 """
-    errorbars(xs, ys, low, high; kwargs...)
-    errorbars(xs, ys, lowhigh; kwargs...)
-    errorbars(points, low, high; kwargs...)
-    errorbars(points, lowhigh; kwargs...)
+    errorbars(pos, low, high; kwargs...)
```

Comment:
For me the most common case (well actually the only) is that `high = ys .+ deltas` and `low = ys .- deltas`. It would be nice if I could just do `errorbars(xs, ys, deltas)`.

Maybe we could have:
```julia
errorbars(xs::Vector{Float}, ys::Vector{Float}, deltas::Vector{Float})
errorbars(xs::Vector{Float}, low_high::Vector{Point2f0})
errorbars(xys::Vector{Point2f0}, deltas::Vector{Float})
```

Also `pos` isn't very clear, it makes me think of `(x, y)` positions. Just `xs` would be more clear imo.

Reply:
I guess `xs` isn't great either since those are either x or y values. Maybe writing out the type would clear this up?

Reply:
I guess the `errorbars(xys::Vector{Point2f0}, deltas::Vector{Float})` signature could work, the first one conflicts with the normal definition, which I'd like to keep as the default

Reply:
the second one could also work

Reply:
It's common to have only one delta for standard deviations etc, but common to have two separate values for any kind of bootstrapping

Reply:
I was thinking of the second one as an alternative to what you have now. I never needed different deltas for low and high, so I'd prefer having `xs, ys, deltas`. Otherwise I'd always have to do the extra work of `lows = ys .- deltas; highs = ys .+ deltas` before plotting.

Reply:
yeah that just depends I guess, I mostly need low and high :)

maybe we can do it this way, with a tuple of vectors option for high low (because it's also a bit annoying having to make points from them):

```julia
# x and y should not flip with direction?
errorbars(x::Vector{Float}, y::Vector{Float}, delta::Vector{Float})

# here val would be either x or y given the direction
errorbars(val::Vector{Float}, high_low::Tuple{Vector{Float}, Vector{Float}})
errorbars(val::Vector{Float}, high_low::Vector{Point2})

# xy should not flip with direction?
errorbars(xy::Vector{Point2}, delta::Vector{Float})
errorbars(xy::Vector{Point2}, high_low::Tuple{Vector{Float}, Vector{Float}})
errorbars(xy::Vector{Point2}, high_low::Vector{Point2})
```

Reply:
although I don't like so much that there is `delta` in one, and `high_low` in the other

Reply:
maybe we need two recipes then, one for delta and one for high/low? maybe `errorbars` for the delta version and `errorranges` for the high-low version

Reply:
or `errorbars` and `rangebars`

Reply:
I would agree with keeping the xy order regardless of `dir`.

> maybe we need two recipes then, one for delta and one for high/low?
That would work as well.